netlify.toml

```diff
@@ -1,5 +1,5 @@
 [build]
-  command = "npm run build"
+  command = "npm run test && npm run build"
```

Comment:
Technically, atm the tests verify fetchy, which is "deployed" as soon as the PR is merged. I think running tests here is a bit unnecessary (and even prevents us deploying even if it's only fetchy that's broken) I probably _wouldn't_ run them here, and rely on the PR check

Reply:
Yup let's do it 👍 I think ultimately unit tests are low cost / run fast - there's no reason not to run them then.